### ISSUE TITLE
Yolov8 Instance Segmentation training doesn't learn anything with Conda Environment on Windows 11

### ISSUE THREAD

**Issue by AhmetEkiz** [ultralytics/ultralytics#3058]
### Search before asking

- [X] I have searched the YOLOv8 [issues](https://github.com/ultralytics/ultralytics/issues) and found no similar bug report.

### Bug

I try to train the instance segmentation model of Yolov8 with Conda on Windows 11. But it doesn't learn anything. 

### Environment

- Windows 11
- Conda
- Torch 2.0.1
- ultralytics 8.0.105

### Minimal Reproducible Example

`yolo segment train data=../datasets/tomato/data.yaml model=yolov8n-seg.pt epochs=10 imgsz=640`

### Additional

```
AMP: running Automatic Mixed Precision (AMP) checks with YOLOv8n...
AMP: checks passed
optimizer: SGD(lr=0.01) with parameter groups 66 weight(decay=0.0), 77 weight(decay=0.0005), 76 bias
train: Scanning C:\Users\ekizo\projects_2\datasets\tomato\train\labels.cache... 800 images, 9 backgrounds, 0 corrupt: 100%|██
val: Scanning C:\Users\ekizo\projects_2\datasets\tomato\valid\labels.cache... 200 images, 1 backgrounds, 0 corrupt: 100%|████
```
 
      Epoch    GPU_mem   box_loss   seg_loss   cls_loss   dfl_loss  Instances       Size
       2/10      2.85G        nan        nan        nan        nan         42        640: 100%|██████████| 50/50 [01:14<00:00
                 Class     Images  Instances      Box(P          R      mAP50  mAP50-95)     Mask(P          R      mAP50  mA
                   all        200        200          0          0          0          0          0          0          0

**Comment by glenn-jocher** [comment]
Hello @AhmetEkiz, thanks for reporting this issue. I'm sorry to hear you are having trouble training the instance segmentation model of YOLOv8 on Conda in Windows 11. However, I'm glad you were able to train the Roboflow YOLOv8 instance segmentation Notebook on Google Colab and that it worked. 

Regarding your issue, a possible reason for the model not learning anything could be because of a mismatch between the dataset and the model. Please verify the following:

- The images and annotations in your dataset have the same filename, and that the annotations are in the COCO JSON format.
- The model architecture you are using (yolov8n-seg.pt) is compatible with the annotations in your dataset. YOLOv8 supports different architectures and datasets, so it's important to choose the architecture that best suits your dataset.
- Verify that the dataset and the annotations are being loaded correctly during training. You could enable the verbose flag to print some debug information during training. 

If none of the above suggestions solve the issue, could you please provide more details such as the size of your training and validation datasets, outputs of the first epoch including the training loss, location of the tensorboard log directory, and the contents of the data.yaml and the class.names files? Please remove all URLs or links, if any. 

Please let us know if you still face the issue, and we will try our best to assist you.

**Comment by AhmetEkiz** [comment]
If I'm not making a mistake, I used the same CLI commands, and the same dataset on Colab and Windows. So, it works and learns on Colab correctly but not on Windows.

**Comment by glenn-jocher** [comment]
We've created a few short guidelines below to help users provide what we need in order to start investigating a possible problem.

## How to create a Minimal, Reproducible Example

When asking a question, people will be better able to provide help if you provide **code** that they can easily understand and use to **reproduce** the problem. This is referred to by community members as creating a [minimum reproducible example](https://stackoverflow.com/help/minimal-reproducible-example). Your code that reproduces the problem should be:

- ✅ **Minimal** – Use as little code as possible to produce the problem
- ✅ **Complete** – Provide all parts someone else needs to reproduce the problem
- ✅ **Reproducible** – Test the code you're about to provide to make sure it reproduces the problem

For [Ultralytics](https://ultralytics.com/) to provide assistance your code should also be:

- ✅ **Current** – Verify that your code is up-to-date with GitHub with `git pull` or `pip install -U ultralytics` to ensure your problem has not already been solved in the latest code version.
- ✅ **Unmodified** – Your problem must be reproducible using official Ultrlaytics code without changes. [Ultralytics](https://ultralytics.com/) does not provide support for custom code ⚠️.

If you believe your problem meets all the above criteria, please close this issue and raise a new one using the 🐛 **Bug Report** [template](https://github.com/ultralytics/ultralytics/issues/new/choose) with a [minimum reproducible example](https://stackoverflow.com/help/minimal-reproducible-example) to help us better understand and diagnose your problem. 

Thank you! 😃

**Comment by AhmetEkiz** [comment]
I will provide more information about this bug. I am not currently available to provide it. So that is why I deleted my previous answer. But I will close this issue. And open with a better one like you said. Thanks for understanding.

**Comment by glenn-jocher** [comment]
Thank you for the update, @AhmetEkiz. We appreciate your willingness to provide more information about the issue. Please feel free to open a new issue when you have more information or any questions in the future. We are always here to help. Have a great day! 😊
